docs/source/reference/northstar/serverdialog.rst

```diff
@@ -0,0 +1,104 @@
+Server dialog
+============
+
+Server dialog is a way for servers to create a dialog window with the option of up to 4 buttons.
+
+Functions
+---------
+
+**Definition:**
+
+.. cpp:function:: void NSSendServerDialog( entity player, ServerDialogData dialog )
+
+Shows the given player the dialog with the data from the struct.
+
+**Example:**
+
+.. code-block:: javascript
```

Comment:
```suggestion
.. code-block::
```
Could you update these codeblocks to be empty? i just merged the lexer so we have proper squirrel highighting as  the default now.